Sources/Exporters/Stdout/StdoutExporter.swift

```diff
@@ -41,3 +51,29 @@ public class StdoutExporter: SpanExporter {
     public func shutdown() {
     }
 }
+
+fileprivate struct SpanExporterData: Encodable {
+    private let span: String
+    private let traceId: String
+    private let spanId: String
+    private let spanKind: String
+    private let traceFlags: TraceFlags
+    private let traceState: TraceState
+    private let parentSpanId: String
+    private let start: Int
+    private let duration: Int
+    private let attributes: [String: AttributeValue]
+    
+    init(span: SpanData) {
+        self.span = span.name
+        self.traceId = span.traceId.hexString
+        self.spanId = span.spanId.hexString
+        self.spanKind = span.kind.rawValue
+        self.traceFlags = span.traceFlags
+        self.traceState = span.traceState
+        self.parentSpanId = span.parentSpanId?.hexString ?? "no Parent"
```

Comment:
```suggestion
        self.parentSpanId = span.parentSpanId?.hexString ?? SpanId.invalid.hexString
```

Reply:
Fixed. I just copied the previous output before.